js/main.js

```diff
@@ -0,0 +1,29 @@
+//Helper functions
+function $q (cssQuery) {
```

Comment:
I realize I wrote `cssQuery` for the argument name, but that's what came to mind at the time. I think that `selector` is argument name that much better represents what you expect.